sp800-63-3/cover.md

```diff
@@ -187,77 +187,78 @@ The terms “SHOULD” and “SHOULD NOT” indicate that among several possibil
 
 The terms “MAY” and “NEED NOT” indicate a course of action permissible within the limits of the publication.
 
-The terms “CAN” and “CANNOT” indicate a possibility and capability, whether material, physical or causal.
+The terms “CAN” and “CANNOT” indicate a possibility and capability, whether material, physical or causal or, in the negative, the absence of that possibility or capability.
 
 <div class="breaker"/>
 
 ## Executive Summary
 
-Digital authentication is the process of establishing confidence in an identity digitally presented to an information system. Authentication establishes that the individual accessing the digital service is who they claim to be, and that this same individual is in control of the technologies used to authenticate.  It provides reasonable assurances that the person that accessed the service yesterday is the same person accessing the service today, and each subsequent time. Digital authentication presents a technical challenge because this process involves the authentication of individual people over an open network to access digital government services, where multiple opportunities for impersonation and other attacks exist.
+Digital identity is the online persona of a subject, and a single definition is widely debated internationally. Persona is apropos as a subject can represent themself online in many ways. An individual may have a digital identity for email, and another one for personal finances.  A personal laptop can be someone's streaming music server yet also be a worker-bot in a distributed network of computers performing complex genome calculations.  Without context, it is difficult to land on a single definition that satisfies all.  Digital identity as a legal identity further complicates the definition and ability to use digital identities across a range of social and economic use cases.  Digital identity is hard.  Proving someone is who they say they are, remotely, via a digital service, is fraught with vulnerabilities of impersonation.  After proving yourself, repeatedly proving it is you logging in is just as complicated and vulnerable as the original claim and proof of identity.  As correctly captured by [Peter Steiner in The New Yorker](#steiner), "On the internet, no one knows you're a dog". These guidelines provide mitigations to the vulnerabilities inherent online, while recognizing and encouraging that when accessing some, low-risk digital services, 'being a dog' is just fine, while other high-risk services need a level of confidence that the digital identity accessing the service is the legitimate proxy to the real life subject.  
```

Comment:
Second sentence start: suggest "The term *persona*..."

"Digital identity as a legal identity" -> "Digital identity as a legal entity"